sql/server.go

```diff
@@ -232,13 +96,13 @@ func (s *Server) exec(req driver.Request, planMaker *planner) driver.Response {
 	return resp
 }
 
-func (s *Server) execStmt(stmt parser.Statement, req driver.Request, planMaker *planner) (driver.Result, error) {
+func (s server) execStmt(stmt parser.Statement, req driver.Request, planMaker *planner) (driver.Result, error) {
```

Comment:
I have a mild preference for leaving `SQL` out of the name. Feels repetitive to have `sql.server.execSQL*`.